metals/src/main/scala/scala/meta/internal/metals/CodeLensProvider.scala

```diff
@@ -70,22 +103,60 @@ final class DebugCodeLensProvider(
           command <- commands
         } yield new l.CodeLens(range, command, null)
       case _ =>
-        Nil
+        Seq.empty
     }
   }
+
+  private def createSuperMethodCommand(
```

Comment:
Let's have a separate code lens providers and just join their results in CodeLensProvider object?

We can have them both take SemanticDB as input. In a similar way as CodeActions